### ISSUE TITLE
Alerts Should be Section 508 Compliant

### ISSUE THREAD

**Issue by pivotal-plech** [pivotal-cf/pivotal-ui#205]
### Current:
![image](https://cloud.githubusercontent.com/assets/4721717/8728393/e05174da-2bb2-11e5-9674-915a1accca19.png)

### Compliant:
![image](https://cloud.githubusercontent.com/assets/4721717/8728338/8f4dbcd8-2bb2-11e5-9921-a9bb259ef542.png)

---

They do have a lot less character, but it's going to be difficult to keep that while providing enough contrast. Perhaps bringing borders back in might give em' a little more oomph.

**Comment by pivotal-plech** [comment]
This is out of date and can be closed. Justin paired with the team and made it so they are both compliant and have a wee bit more character :)

![image](https://cloud.githubusercontent.com/assets/4721717/9481240/8638b42c-4b56-11e5-8663-a09ce4ca776c.png)
